thornodes/alerting.md

```diff
@@ -1,255 +1,47 @@
-
-[https://github.com/block42-blockchain-company/thornode-telegram-bot](https://github.com/block42-blockchain-company/thornode-telegram-bot)
-{% endhint %}
-
-### Requirements
-
-* Telegram
-* Docker \(if you want to run with docker or docker-compose\)
-* Docker Compose \(if you want to run with docker-compose\)
-* Python3 \(if you want to run without docker\)
-
-### Quickstart
-
-Open `variables.env` file and set
-
-* `TELEGRAM_BOT_TOKEN` to your Telegram Bot Token obtained from BotFather.
-* `THORCHAIN_NODE_IP` to any THORNode IP you want to monitor \(or `localhost`\). Leave it empty or remove it to use testnet seed Node IPs.
-* `BINANCE_NODE_IP` to any Binance Node IP you want to monitor \(or `localhost`\). Leave it empty or remove it to not monitor a Binance Node.
-* `ADMIN_USER_IDS` to a list of Telegram User IDs that are permissioned to access the Admin Area.
-
-Install `docker` and `docker-compose` and run:
-
-```text
-docker-compose up -d
-```
-
-### Steps to run everything yourself
-
-### [Install dependencies](https://github.com/block42-blockchain-company/thornode-telegram-bot/issues/new#install-dependencies)
-
-Install all required dependencies via: `pip install -r requirements.txt`
-
-### [Create Telegram bot token via BotFather](https://github.com/block42-blockchain-company/thornode-telegram-bot/issues/new#create-telegram-bot-token-via-botfather)
-
-Start a Telegram chat with [BotFather](https://t.me/BotFather) and click `start`.
+## Vǫrðr 👻
 
-Then send `/newbot` in the chat, and follow the given steps to create a new telegram token. Save this token, you will need it in a second.
+A monitoring application for THORNodes.
 
-### [Set environment variables](https://github.com/block42-blockchain-company/thornode-telegram-bot/issues/new#set-environment-variables)
-
-Set the telegram bot token you just created as an environment variable: `TELEGRAM_BOT_TOKEN`
-
-```text
-export TELEGRAM_BOT_TOKEN=XXX
-```
-
-Next you can specify the IP of the Thornode that you want to watch in the `THORCHAIN_NODE_IP` environment variable.
-
-Set it to `localhost` to listen a node on your own machine.
-
-Leave this environment variable empty or don't even set it to use IPs of the testnet seed nodes from [https://testnet-seed.thorchain.info](https://testnet-seed.thorchain.info/).
+> A Vǫrðr or warden is a guardian spirit who will follow the soul of a living person from birth until death.
 
 {% hint style="info" %}
-_Please note, if you leave `THORCHAIN_NODE_IP` empty, IP specific monitoring won't take effect \(no check for increasing block height, midgard API and catch up status\). Because we rotate through available testnet seed IPs, we would compare data of different nodes and send incorrect alerts._
+Vǫrðr is a community developed monitoring app for THORNodes. It's fully open-source at [https://github.com/sourcapital/vordr](https://github.com/sourcapital/vordr).
```

Comment:
My 2 sats - this is fine as long as it is very clear that this is 3rd party software outside of the scope of THORSec.

Reply:
I added an extra disclamer that said that.